cfme/infrastructure/deployment_roles.py

```diff
@@ -231,7 +231,10 @@ def provider(self):
 
     def delete(self, cancel=False):
         view = navigate_to(self, 'Details')
-        view.toolbar.configuration.item_select('Remove item',
+        remove_item = VersionPick({
```

Comment:
**REQUIRED**
`VersionPick` is not supposed to be used not in view. Please use such construction:
```python
if self.appliance.version < '5.9':
    'Remove item'
else:
    'Remove item from Inventory'
```